main.py

```diff
@@ -0,0 +1,53 @@
+import sys
+from PIL import Image
+from pathlib import Path
+from random import randint
+
+
+def getimages(location):
+    image = []
+    validimages = [".jpg", ".jpeg", ".png"]
+    p = Path(location)
+    try:
+        for x in p.iterdir():
+            ext = Path(x).suffix
+            if x.is_file() and ext in validimages:
+                image.append(x)
+    except FileNotFoundError as e:
+        print(e)
+        print("No such file exists")
+        sys.exit()
+    except Exception as e:
+        print(e)
+        print("There was a problem")
+        sys.exit()
+    else:
```

Comment:
beautiful usage of else block after try-except. not many people use it, but personally, I like these feature.

Reply:
:)